README.md

```diff
@@ -1,3 +1,37 @@
 # node-osm-api
 
-easy communication with osm api
+Easy nodejs communication with OpenStreetMap api
+
+```
+import Apiv6 from '@map-colonies/node-osm-api';
+
+const api = new Apiv6('https://www.openstreetmap.org', USER_NAME, PASSWORD);
+```
+
+## Installation
+
+```
+$ npm install @map-colonies/node-osm-api
+```
+
+## Usage
+
+- Right now we only support create/upload/close changesets
+
+Create changeset:
+
+```
+api.createChangeset(XML_AS_STRING).then(console.log).catch(console.error);
+```
```

Comment:
`.then(console.log).catch(console.error)` is not very informative, 

what is the result of the function?
which errors can be thrown if something goes wrong?

Reply:
done